tasks/infra/do-custom-updates.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: do-custom-updates
+spec:
+  description: 'Allow for patching custom resources and other custom updates'
+  params:
+    - name: kubeconfig-path
+      type: string
+  steps:
+    - name: do-custom-updates
+      computeResources:
+        limits:
+          cpu: '100m'
+          memory: 64Mi
+      image: quay.io/kuadrant/testsuite-pipelines-tools:latest
+      imagePullPolicy: Always
+      env:
+        - name: KUBECONFIG
+          value: $(params.kubeconfig-path)
+      script: |
+        #!/bin/bash
+        set -euo pipefail
+
+        # Get the namespace of the Subsciption (typically kuadrant-system but it can be different)
+        namespace=$(kubectl get subscription --all-namespaces -o json | jq -r '.items[] | select(.metadata.name == "kuadrant-operator") | .metadata.namespace')
+        if [[ -z "$namespace" ]]; then
+            echo "Subcription 'kuadrant-operator' not found in any namespace."
+            exit 1
+        fi
+
+        # Limitador log level to DEBUG
+        kubectl patch limitador limitador --namespace $namespace --type merge --patch '{"spec":{"verbosity":3}}'
+        # Limitador disk storage
+        kubectl patch limitador limitador --namespace $namespace --type merge --patch '{"spec":{"storage":{"disk":{}}}}'
```

Comment:
I am unsure about setting this by default, is there some reason for it? The testsuite expects the storage option to be off and turns it on with disk, redis and redis-cache storages as part of disruptive tests. And after merging of GRL test it will set it as part of the test case.
In normal testing the limitador pod does not need durability. Also adding this by default would complicate upgrade testing due to https://github.com/Kuadrant/limitador-operator/issues/196

I suggest not turning this on by default